src/test/java/org/codehaus/mojo/exec/ExecMojoTest.java

```diff
@@ -356,4 +357,82 @@ private String getCommandLineAsString(CommandLine commandline) {
         }
         return result;
     }
+
+    public void testGetShebang() throws Exception {
+        ExecMojo execMojo = new ExecMojo();
+        Method method = ExecMojo.class.getDeclaredMethod("getShebang", File.class);
+        method.setAccessible(true);
```

Comment:
Instead of reflection we can make method `getShebang` package private, test will be clear